tsconfig-common.json

```diff
@@ -26,6 +26,23 @@
 
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        // Setting `skipLibCheck: true`, avoids the following issues in various .d.ts files:
+        // - node_modules/@google-cloud/kms/node_modules/long/index.d.ts:446:1 - error
+        //   TS1203: Export assignment cannot be used when targeting ECMAScript modules.
+        //   Consider using 'export default' or another module format instead.
```

Comment:
This one is not a conflict.  Maybe the .d.ts files will be different in a more recent version.  I don't want to make changes that could modify the runtime behavior.

Reply:
**praise:** Thanks. This is really helpful. It won't catch any new issues that come up, but it gives us something to understand.